Dockerfile

```diff
@@ -11,19 +11,24 @@
 
 FROM golang:1.12-alpine as builder
 USER root
-RUN apk add curl git
+RUN apk add curl git nodejs npm
```

Comment:
node@10 and npm@6 for consistency
we can look at newer versions once everything's up and running